common/sgx/collateral.h

```diff
@@ -66,6 +66,16 @@ oe_result_t oe_get_sgx_quote_verification_collateral(
 void oe_free_sgx_quote_verification_collateral_args(
     oe_get_sgx_quote_verification_collateral_args_t* args);
 
+/**
+ * Pre-allocate memory for the resources that get passed into the 
+ * oe_get_quote_verification_collateral_ocall. To avoid extra passes needed
+ * to take care of OE_BUFFER_TOO_SMALL failures.
+ *
+ * @param[in] args The quote verification collateral.
+ */
+void oe_prealloc_quote_verification_collateral_args(
```

Comment:
You can just forward declare this in in the .c file, it doesn't need to be in the header

Reply:
I agree, I'll move it to the .c file. 

My initial intent in adding it to the header was:
1. to maintain consistency with other functions defined here like:  oe_free_sgx_quote_verification_collateral_args() and oe_get_sgx_quote_verification_collateral()
2. If this function ever needs to be repurposed. Which currently doesn't look like the case.